packages/twenty-server/src/engine/core-modules/analytics/analytics.service.ts

```diff
@@ -25,28 +29,45 @@ export class AnalyticsService {
     workspaceDomainName: string | undefined,
     hostName: string | undefined,
   ) {
-    if (!this.environmentService.get('TELEMETRY_ENABLED')) {
+    if (this.environmentService.get('ANALYTICS_ENABLED')) {
+      const data = {
+        action: createEventInput.type,
+        timestamp: new Date().toISOString(),
+        version: '1',
+        session_id: createEventInput.sessionId,
+        payload: {
+          userId: userId,
```

Comment:
If data within the payload is in camelCase (which make sense, it's what we usually use) then you should adapt your tinybird data model to also be in snakeCase (sessionId not session_id)